hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -263,7 +286,7 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
     tableLevelMutex.lock();
     HiveLockHeartbeat hiveLockHeartbeat = null;
     try {
-      lockId = Optional.of(acquireLock());
+      lockId = Optional.ofNullable(acquireLock(agentInfo));
```

Comment:
Here I get an Intellij warning because acquireLock returns long which is never null , should we keep Optional.of()?